docs/3.1/docs/tsa-arch/why-daml.rst

```diff
@@ -0,0 +1,98 @@
+
+Enterprise Requirements: Privacy and Independent Control
+********************************************************
+Daml is targeted at enterprise use cases. App providers and app users are enterprises that operate in a regulatory regime, which imposes constraints on the technologies used by these enterprises. For financial services companies there are two properties that a technology must afford in business transactions to be able to comply with the regulations.
+
+Privacy
+=======
+In financial and other regulated industries, privacy—specifically sub-transaction privacy—is crucial. In a tokenization application, confidentiality around assets and transfers must be maintained at a subtransaction level. For example, it is not acceptable for the cap table of a tokenized financial asset to be publicly visible, nor should transfers be visible to parties not involved in the transaction.
+
+Daml addresses this need for privacy through the Canton protocol, which decomposes transaction data into views or projections. These views allow stakeholders to see only the information relevant to their role, as defined in the smart contracts. This sub-transaction privacy ensures confidentiality by distributing sensitive data on a strict need-to-know basis to authorized parties.
+
+Design Considerations to Ensure Privacy
+---------------------------------------
+To ensure privacy, Daml contracts must be designed in a way that models who gets to see which data. Appropriate roles should be assigned to stakeholders within the contract, ensuring that sub-transaction privacy is maintained in compliance with regulatory requirements.
+
+Independent Control
+===================
+The second key requirement is independent control. Issuers and custodians of digital assets have regulatory and fiduciary responsibilities to control the availability, scalability, and performance of the technology employed for tokenized asset transactions. They also need to comply with regulatory requirements such as Know Your Customer (KYC) and Anti-Money Laundering (AML). Therefore, they need to control the access to assets and transactions. They need control over the smart contracts for any special scenarios that need corrections or overriding. Finally, the technology must be able to evolve to continue to comply with new regulations.
+
+Architectural Considerations to Ensure Independent Control
+----------------------------------------------------------
+For these reasons, app users must have full control over their legal system of records. If an app user relies solely on the app provider’s participant node, reconciliation challenges may resurface, since it would need to integrate its system of records with the app provider’s participant node. Instead, the app user should deploy their own participant node that serves as their primary source of data to maintain data sovereignty and independent control.
```

Comment:
```suggestion
For these reasons, app users must have full control over their legal system of records. If an app user relies solely on the app provider’s participant node, it would need to integrate its system of records with that node, potentially recreating reconciliation challenges. Instead, the app user should deploy its own participant node that serves as its primary data source to maintain data sovereignty and independent control.
```